pkgs/development/python-modules/rtp/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python3
+
+# buildInputs
+, mypy
+, hypothesis
+
+# propagatedBuildInputs
+, six
+, flask
+}:
+
+buildPythonPackage rec {
+  pname = "rtp";
+  version = "0.0.2";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-xUfI8lQD/EhA4NrE4y+KHlie+wwBWkYkDqFLVWKwOzA=";
+  };
+
+  buildInputs = [
+    mypy
+    hypothesis
+  ];
+
+  propagatedBuildInputs = [
+    six
+    flask
+  ];
+
+  pythonImportsCheck = [
+    "rtp"
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+    ${python3.interpreter} -m unittest discover -s tests
+    runHook postCheck
+  '';
```

Comment:
```suggestion
```

Reply:
Just adding `unittestCheckHook` didn't worked therefore I went this way, but just learned about `unittestFlagsArray` and this works now without changing the checkPhase manually :)